files/es/learn/common_questions/how_does_the_internet_work/index.html

```diff
@@ -80,9 +80,9 @@ <h3 id="Una_red_de_redes">Una red de redes</h3>
 
 <h3 id="Encontrando_ordenadores">Encontrando ordenadores</h3>
 
-<p>Si deseas enviar un mensaje a una computadora, debes especificar a cuál. Es por ello que todo ordenador conectado a una red cuenta con una dirección única que lo identifica, llamada “dirección IP” o Protocolo de Internet(IP de sus siglas en inglés <em>Internet Protocol</em>). Esta dirección está compuesta por una serie de cuatro números separados por puntos, por ejemplo: <code>192.168.2.10</code>.</p>
+<p>Si deseas enviar un mensaje a una computadora, debes especificar a cuál. Es por ello que todo ordenador conectado a una red cuenta con una dirección única que lo identifica, llamada “dirección IP” (siendo "IP" las siglas en inglés de <em>Internet Protocol</em>, o <em>Protocolo de Internet</em>). Esa dirección está compuesta por una serie de cuatro números separados por puntos, por ejemplo: <code>192.168.2.10</code>.</p>
 
-<p>Para los ordenadores es un identificador simple, pero los humanos tienen mayor dificultad a la hora de recordar y memorizar este tipo de dirección. Con el propósito de convertir esta serie numérica en algo que podamos asociar con mayor facilidad a la dirección IP se utiliza lo que conocemos como <em>nombre de dominio</em>. Por ejemplo, <code>google.com </code>es el nombre de dominio utilizado para sustituir la dirección IP <code>173.194.121.32</code>. Así, usar un nombre de dominio es la manera más fácil para nosotros de identificar un ordenador a través de Internet.</p>
+<p>Para los ordenadores es un identificador simple, pero los humanos tenemos dificultades para recordar este tipo de direcciones. Para facilitarnos las cosas, es posible asociar una dirección IP con una denominación más legible llamada <em>nombre de dominio</em>. Por ejemplo, <code>google.com</code> es el nombre de dominio utilizado para la dirección IP <code>173.194.121.32</code>. Así, usar un nombre de dominio es la manera más fácil para nosotros de identificar un ordenador a través de Internet.</p>
```

Comment:
* "dificultades" habla de más de alguna dificultad
* ¿qué cosas? usar redacciones claras, menos informales, pero entendibles

```suggestion
<p>Para los ordenadores es un identificador simple, pero los humanos tenemos mayor dificultad para recordar este tipo de direcciones. Con el propósito de convertir esta serie numérica en algo que podamos asociar con mayor facilidad a la dirección IP se utiliza lo que conocemos como <em>nombre de dominio</em>. Por ejemplo, <code>google.com</code> es el nombre de dominio utilizado para la dirección IP <code>173.194.121.32</code>. Así, usar un nombre de dominio es la manera más fácil para nosotros de identificar un ordenador a través de Internet.</p>
```

Reply:
Respecto de dificultad o dificultades, no estoy tan seguro... En cuanto a lo de "Para facilitarnos las cosas", la verdad es que me quedé pegado a la versión en inglés, "To make things easier". Probablemente no tenga mucho sentido agregar eso al texto en castellano, aunque la expresión ("Para facilitarnos las cosas") es válida en el castellano que uso habitualmente... Gracias por revisar lo que escribí...!

Reply:
En el caso de lo de la versión en inglés, ya tocaría reformular el párrafo entero y de ahí ya sería mejor pasar a actualizar el documento entero.